utils/src/api_schema/api_schema.json

```diff
@@ -173,6 +175,28 @@
       "additionalProperties": false
     },
 
+    "operationFlip": {
+      "type": "object",
+      "properties": {
+        "type":   { "enum": [ "flip" ] },
+        "code":   { "type": "integer" }
+      },
+      "required": ["type", "code"],
+      "additionalProperties": false
+    },
+
+    "operationRotate": {
```

Comment:
Could you please make sure the wiki page is updated to reflect this. I haven't checked if it is